tools/who_what_benchmark/whowhatbench/wwb.py

```diff
@@ -467,8 +269,21 @@ def genai_gen_image(model, prompt, num_inference_steps, generator=None):
     return image
 
 
+def genai_gen_image2image(model, prompt, image, num_inference_steps, generator=None):
+    image_data = ov.Tensor(np.array(image.getdata()).reshape(1, image.size[1], image.size[0], 3).astype(np.uint8))
+    image_tensor = model.generate(
+        prompt,
+        image=image_data,
+        num_inference_steps=num_inference_steps,
+        strength=0.8,
```

Comment:
do you run optimum / diffusers with the same `strength` value?

Reply:
Good point. It is 0.8 by default in Diffusers but it is better to set it explicitly.

Reply:
Actually, it is already set to 0.8 as well in default_gen_image_fn of im2im pipeline.